roles.json

```
@@ -10,21 +10,33 @@
-        "responsibilities": {
-            "description": "Overall coordination and management of the Astropy project, including:",
-            "details": [
-		"Formally detailed in the <a href='https://github.com/astropy/astropy-APEs/blob/main/APE0.rst#the-coordination-committee'>The Astropy Project Governance Charter (APE 0)</a>",
-                "Keeping a large-scale view of the Astropy ecosystem",
-                "Evaluating new affiliated package submissions and review of existing affiliated packages",
-                "Approving or rejecting Astropy APEs",
-                "Evaluating and merging core package pull requests as needed (e.g., for sub-packages without a maintainer)",
-                "Arbitrating disagreements in the core package, including final decisions when otherwise deadlocked",
-                "Maintaining the list of roles and related github permissions",
-                "Managing finances for the project",
-                "Coordinating with NumFOCUS and other funding organizations",
-                "Securing funding for the project via discussions and proposals to funding agencies"
-            ]
-        }
+        "responsibilities": [
+            {
+                "description": "Overall coordination and management of the Astropy project, including:",
+                "details": [
+		            "Formally detailed in the <a href='https://github.com/astropy/astropy-APEs/blob/main/APE0.rst#the-coordination-committee'>The Astropy Project Governance Charter (APE 0)</a>",
+                  "Keeping a large-scale view of the Astropy ecosystem",
+                  "Evaluating new affiliated package submissions and review of existing affiliated packages",
+                  "Approving or rejecting Astropy APEs",
+                  "Evaluating and merging core package pull requests as needed (e.g., for sub-packages without a maintainer)",
+                   "Arbitrating disagreements in the core package, including final decisions when otherwise deadlocked",
+                   "Maintaining the list of roles and related github permissions",
+                   "Managing finances for the project",
+                   "Coordinating with NumFOCUS and other funding organizations",
+                   "Securing funding for the project via discussions and proposals to funding agencies"
+                 ]
+            },
+            {
+                "description": "Organisation and communication:",
+                "details": [
+                    "Policies: <a href='https: //github.com/astropy/astropy-project/tree/main/policies/coco-operating-policies.md'>CoCo Operating Policies</a>.",
+                    "Meeting records: <a href='https://docs.google.com/document/u/0/d/1kKzSVTom27HiWaw5yTGZDL7vLVAjBGhueXAl2wuSlMY/edit'>CoCo Running Notes</a>.",
+                    "Communication within the coordination committee happens through regular zoom meetings and a private Slack channel.",
```

```suggestion
                    "Communication within the coordination committee happens through regular Zoom meetings and a private Slack channel.",
```
absolutely trivial suggestion, but Slack is capitalized so Zoom should be too